client/my-sites/checkout/composite-checkout/lib/get-jetpack-product-features.ts

```diff
@@ -0,0 +1,20 @@
+import { isJetpackBackupT1Slug } from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+import type { WithSnakeCaseSlug } from '@automattic/calypso-products';
+
+export default function getJetpackProductFeatures(
+	product: WithSnakeCaseSlug,
+	translate: ReturnType< typeof useTranslate >
+): string[] {
+	if ( isJetpackBackupT1Slug( product.product_slug ) ) {
```

Comment:
Would it be more reusable to make a more generic lib function `isJetpackProductSlug( JETPACK_BACKUP_T1_PRODUCTS, product.product_slug )` instead of defining the function explicitly for Jetpack Backup?

Reply:
I think the reason there isn't one of those yet is potentially to avoid having to import the constants directly from the package and keeping it more contained like that.

The idea of this `isProductSlug` wouldn't really have to do anything with the Calypso packages since you'd essentially be giving it a list of products, the product slug, and asking it if the second parameter is in the array of the first. That could just be re-written as something like this
```
if( JETPACK_BACKUP_T1_PRODUCTS.includes( product.product_slug ) ) {
...
}
```

Which is not the end of the world, but I think I like keeping these functions set in the `calypso-products` package and not to break the consistency here

Reply:
That's fair; thanks for explaining it! I'm fine with having it like that then